daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md

```diff
@@ -98,6 +98,22 @@ logging:
     enabled: true
 ```
 
+### Obfuscate URLs in HTTP API logging
+
+By default, logs for API calls in the HTTP endpoints include the full URL being invoked, for example `POST /v1.0/invoke/directory/method/user-123`, which could contain Personal Identifiable Information (PII).
+
+To reduce the risk of PII being accidentally included in API logs (when enabled), Dapr can instead log the abstract route being invoked, for example `POST /v1.0/invoke/{id}/method/{method:*}`. This can help ensuring compliance with privacy regulations such as GDPR.
```

Comment:
```suggestion
To reduce the risk of PII being accidentally included in API logs (when enabled), Dapr can instead log the abstract route being invoked, for example `POST /v1.0/invoke/{id}/method/{method:*}`. This can help ensuring compliance with privacy regulations such as GDPR.
```